File: frontend/src/algorithmCode/searching/linearSearch/linearSearch.java
public class LinearSearch {
    public static int linearSearch(int[] arr, int target) {
        for (int i = 0; i < arr.length; i++) {
            if (arr[i] == target)
                return i; // Found
        }
        return -1; // Not found
    }

    public static void main(String[] args) {
        int[] arr = {3, 5, 7, 9, 11};
        int target = 9;
        int result = linearSearch(arr, target);

        if (result != -1)
            System.out.println("Element found at index " + result);
        else
            System.out.println("Element not found");
    }
}

File: frontend/src/algorithmCode/searching/linearSearch/linearSearch.py
def linear_search(arr, target):
    for i in range(len(arr)):
        if arr[i] == target:
            return i  # Found
    return -1  # Not found

# Example usage
arr = [3, 5, 7, 9, 11]
target = 9
result = linear_search(arr, target)

if result != -1:
    print(f"Element found at index {result}")
else:
    print("Element not found")
